### PR TITLE
Jbaker/update demo components/#12

### PR DESCRIPTION
**Pull request by jason-capsule42**
# Alaska Airlines Pull Request

Resolves: #12

## Summary:

Add documentation around custom element best practices.

## Type of change:

Please delete options that are not relevant.

- [x] Update to all current Auro Components and new content design

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,88 +1,116 @@
...
+| npm start | Runs the app in the development mode.<br />Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+||Browser sync and linting in the CLI is enabled.
+| npm test | Launches the test runner in the interactive watch mode
+|| See [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+| npm build | Production build in the `build` folder.<br>Optimized, minimized and prepared for deployment
+||See [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+
 
-## Setting up a new Svelte project with Auro Web Components
+## Setting up new React project with Auro Web Components
```
Comment (geoffrich) [inline README.md:25]: Should be "Svelte", not "React"

File: README.md [hunk 2]
```diff
@@ -1,88 +1,116 @@
...
 
-The following steps will let you start using Web Components in your Svelte application across all [supported browsers](https://auro.alaskaair.com/support/browsersSupport).
+The following steps will let you start using Web Components in your React application across all supported browsers.
```
Comment (geoffrich) [inline README.md:27]: Should be "Svelte", not "React"

File: rollup.config.js [hunk 3]
```diff
@@ -43,8 +43,6 @@ export default {
     copy({
       copyOnce: true,
       targets: [
-        { src: 'node_modules/@webcomponents/webcomponentsjs/bundles/*', dest: 'public/build/webcomponentsjs/bundles'},
-        { src: 'node_modules/@webcomponents/webcomponentsjs/webcomponents-loader.js', dest: 'public/build/webcomponentsjs'},
         { src: 'node_modules/shimport/index.js', dest: 'public/build/shimport'}
```
Comment (geoffrich) [inline rollup.config.js:46]: We also don't need shimport if we're dropping IE

File: public/index.html [hunk 4]
```diff
@@ -13,8 +13,6 @@
 </head>
 
 <body>
-	<script src="https://polyfill.io/v3/polyfill.min.js?features=URL%2CArray.from%2Cfetch"></script>
-	<script src="./build/webcomponentsjs/webcomponents-loader.js"></script>
```
Comment (geoffrich) [inline public/index.html:17]: Replace the shimport function with a 
```
<script type="module" src="./build/main.js"></script>
```

File: src/App.svelte [hunk 5]
```diff
@@ -1,85 +1,146 @@
...
+			'value': 'canada'
+		},
+		{
+			'label': 'Mexico',
+			'value': 'mexico'
 		}
 	];
 
-	$: selectedOptions = options.filter(option => option.checked).map(option => option.value);
+	function calcDestinations(e) {
```
Comment (geoffrich) [inline src/App.svelte:40]: I think there's a more Svelte-y way to write this stuff. Mind if I take a stab at it?

File: src/global.scss [hunk 6]
```diff
@@ -1,10 +1,25 @@
...
+@import '../node_modules/@alaskaairux/webcorestylesheets/dist/normalize';
+@import '../node_modules/@alaskaairux/webcorestylesheets/dist/essentials';
+@import '../node_modules/@alaskaairux/webcorestylesheets/dist/headings';
+
+main {
+  background-color: var(--auro-color-background-lightest);
+  padding: var(--auro-size-xl);
+}
+
+.formDataWrapper {
```
Comment (geoffrich) [inline src/global.scss:17]: We should move these styles into the component so they're scoped.

**Review by geoffrich — COMMENTED** [review]
There's an opportunity to refactor this to be more idiomatic to Svelte. Do you mind if I try to refactor and push to your branch @jason-capsule42 ?

**Comment by geoffrich** [comment]
I pushed a commit making the code more Svelte-idiomatic. @jason-capsule42 let me know if you have questions about any of my changes.

**Comment by blackfalcon** [comment]
@geoffrich is doing a great job with this review. Carry on!

**Comment by jason-capsule42** [comment]
@geoffrich I finished updates based on your initial review comments. If you could take a final pass on this so we can get it out the door that would be rad.

File: README.md [hunk 7]
```diff
@@ -1,88 +1,114 @@
-# Svelte Demo
+# Web Component Svelte Demo
 
 ![Travis (.org)](https://img.shields.io/travis/alaskaairlines/AuroSvelteDemo?style=for-the-badge) ![GitHub tag (latest by date)](https://img.shields.io/github/v/tag/AlaskaAirlines/AuroSvelteDemo?style=for-the-badge)
 
-Interested in starting a new project using Auro and Svelte? This demo project is a complete example that includes basic setup. 
+An example Svelte project with Auro Components integrated. This app runs and works in all Alaska-supported browsers. Explore the project source to see the Auro Components being used in a frameworkless environment.
```
Comment (geoffrich) [inline README.md:5]: "frameworkless environment" doesn't apply here. Maybe this was copied from the vanilla JS demo?

File: README.md [hunk 8]
```diff
@@ -1,88 +1,114 @@
...
-```bash
-$ git clone https://github.com/AlaskaAirlines/AuroSvelteDemo.git
-```
-
-The app runs and works in all [supported browsers](http://auro.alaskaair.com/support/browsersSupport). Please explore the project source to see how the Auro components are being used in a Svelte development environment. 
-
-This is a project template for [Svelte](https://svelte.dev) apps. It lives at [https://github.com/sveltejs/template](https://github.com/sveltejs/template).
-
-To create a new project based on this template using [degit](https://github.com/Rich-Harris/degit):
+Below are instructions for adding Auro compatibility to your project. A familiarity with modern Javascript development and bundlers such as Webpack is expected.
```
Comment (geoffrich) [inline README.md:7]: I wouldn't mention Webpack here. Maybe take out the second sentence entirely. That was a sentence from the vanilla JS demo which requires more wiring up bundlers and whatnot.

File: package.json [hunk 9]
```diff
@@ -25,31 +24,25 @@
...
-    "rollup-plugin-svelte": "^5.2.3",
+    "rollup-plugin-svelte": "^6.1.1",
     "rollup-plugin-terser": "^5.1.2",
     "semantic-release": "^17.1.1",
     "shimport": "^1.0.1",
     "svelte": "^3.24.0",
     "svelte-preprocess": "^3.9.12"
   },
   "dependencies": {
+    "@alaskaairux/webcorestylesheets": "^3.4.0",
```
Comment (geoffrich) [inline package.json:43]: I think this should be a dev-dep.

File: README.md [hunk 10]
```diff
@@ -1,88 +1,114 @@
...
 
 | Command | Description
 |---|---
-| npm start | Runs the app in the development mode.<br />Open [http://localhost:5000](http://localhost:5000) to view it in the browser.
-| npm run build | Production build in the `build` folder.
-| npm run dev | Runs dev server with browser sync enabled.
+| npm start | Runs the app in the development mode.<br />Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+||Browser sync and linting in the CLI is enabled.
+| npm test | Launches the test runner in the interactive watch mode
+| npm build | Production build in the `build` folder.<br>Optimized, minimized and prepared for deployment
```
Comment (geoffrich) [inline README.md:20]: We should remove `npm test` since we don't have any tests and also mention `npm run dev`.

File: README.md (hunk 7 above)
Comment (jason-capsule42) [reply README.md:5]: Yes, yes it was. I'll remove the reference.

File: README.md (hunk 8 above)
Comment (jason-capsule42) [reply README.md:7]: Good catch. Removed.

File: package.json (hunk 9 above)
Comment (jason-capsule42) [reply package.json:43]: Moved.

**Review by geoffrich — APPROVED** [review]

**Pull request merged**